base/bitset.jl

```diff
@@ -16,6 +16,29 @@ mutable struct BitSet <: AbstractSet{Int}
     offset::Int
 
     BitSet() = new(sizehint!(zeros(UInt64, 0), 4), NO_OFFSET)
+    function BitSet(v::AbstractVector)
+        isempty(v) && return new(zeros(UInt64, 0), NO_OFFSET)
```

Comment:
Should this include the sizehint? Maybe
```suggestion
        isempty(v) && return BitSet()
```

Reply:
I don't think it should include a size hint. I think the primary reason for a size hint in the generic iterable case is because creating a BitSet with `BitSet(itr)` involves making an empty bitset and then pushing into it. We reach this case when someone ruIns `BitSet(Int[])` which may well never have new elements added to it.